src/Download.java

```diff
@@ -107,47 +108,34 @@ static void main(boolean dryRun, String... args) {
     }
   }
 
-  static class JDK {
-
-    final String release;
-    final String version;
-    final String os;
-    final String arch;
-    final String type;
-
-    JDK(String release, String version, String os, String arch, String type) {
-      this.release = release;
-      this.version = version;
-      this.os = os;
-      this.arch = arch;
-      this.type = type;
-    }
+  record JDK(String release, String version, String os, String arch,
+             String type) {
 
     @Override
-    public String toString() {
+      public String toString() {
```

Comment:
Delete the custom `toString()` method. The one generated by `record` is fine.

Reply:
fixed